packages/templates/react-urql-graphql-codegen/src/components/str/listPageTemplate.ts

```diff
@@ -0,0 +1,22 @@
+const listPageTemplate = `import { useGeneratedQuery } from '../generated'
```

Comment:
this is a duplicity with https://github.com/iteria-app/lowcode/blob/41bee8138b93a36997117a2abefdfc7d7ef6ca62/packages/templates/react-urql-graphql-codegen/src/components/App.tsx (we need to come up with proper way of importing without this kind of duplicity - and there is much more than this file in that package: package.json deps ...).